modules/mauth-signer/src/main/java/com/mdsol/mauth/DefaultSigner.java

```diff
@@ -80,4 +114,45 @@ public DefaultSigner(UUID appUUID, PrivateKey privateKey, EpochTimeProvider epoc
 
     return headers;
   }
+
+  @Override
+  public Map<String, String> generateRequestHeadersV2(String httpVerb,
+      String requestPath, String requestPayload, String queryParameters) throws MAuthSigningException {
+
+    if (disableV1 == false && mauthVersion.equalsIgnoreCase(MAuthVersion.MWS.getValue())) {
+      return generateRequestHeaders(httpVerb, requestPath, requestPayload);
```

Comment:
quick question!
when `disableV1` is false, do you have both V1 and V2 headers in the request headers?

https://github.com/mdsol/mauth-client-ruby/blob/v5.0.2/spec/client/signer_spec.rb#L20-L23

Reply:
If disableV1 is false and the specified version is MWS(V1), only has V1 headers. Otherwise, will generate V2 headers only.

Reply:
During the transition phase, we need to support the both versions so I think you need to modify the code to have both headers when disableV1 is false.

Reply:
Fixed in https://github.com/mdsol/mauth-jvm-clients/pull/89/commits/51b34b1f9bbc72ec164f516b827fa41eeafd9852